core/src/pages/community.astro

```diff
@@ -184,7 +184,7 @@ const widerEcosystem = [
   },
   {
     href: 'https://determinate.systems/nix/',
-    name: 'Determinate Nix',
+    name: 'Determinate Systems fork',
```

Comment:
```suggestion
    name: 'Determinate Systems\' Nix fork',
```

Sorry for the nit, but the grammar threw me off and took me some attempts to parse. Moreover, making clear *what* is being forked is relevant information IMO, which most people knowing about Determinate Nix have but new people to the ecosystem whom this page targets don't